datahub-frontend/app/controllers/AuthenticationController.java

```diff
@@ -292,7 +292,8 @@ private Optional<Result> redirectToIdentityProvider(Http.RequestHeader request,
 
     private void configurePac4jSessionStore(PlayWebContext context, Client client, String redirectPath) {
         // Set the originally requested path for post-auth redirection.
-        _playSessionStore.set(context, Pac4jConstants.REQUESTED_URL, new FoundAction(redirectPath));
+        // Commenting this to avoid https://github.com/datahub-project/datahub/issues/7221
```

Comment:
Let me confirm with my team if removing this might have any side effects and get back to you

Reply:
Sure, thank you for taking a look! The only drawback I see for this change is that shared links to resources in Datahub will only work for authenticated users, otherwise once the it gets back from the OIDC provider it'll land at the home page. I the user hits the link again it works. I believe this was feature added on [v0.9.6.1](https://github.com/datahub-project/datahub/releases/tag/v0.9.6.1)